pyproject.toml

```
@@ -10,8 +10,10 @@ readme = "README.md"
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = ">=3.11,<3.13"
+python = ">=3.10,<3.12"
 hydra-core = "^1.3.2"
+apache-beam = {extras = ["gcp"], version = "^2.51.0"}
```

We don't have any `gcp` extra defined?

```suggestion
apache-beam = "^2.51.0"
```

---

`apache-beam[gcp]` is needed.

---

Still relevant question?

---

I am not sure what you are asking... Doing `poetry add apache-beam[gcp]`, yields `apache-beam = {extras = ["gcp"], version = "^2.51.0"}`?

---

Oooh I'm misunderstanding here, I thought this _added_ a new extra `gcp`, so that we could `pip install tts_text[gcp]`, say. But it installs `apache_beam` with the `gcp` extra, I see!